cdflib/cdfread.py

```diff
@@ -1739,7 +1739,7 @@ def _read_data(
 
         # Put the data into system byte order
         if self._convert_option() != "=":
-            ret = ret.byteswap().newbyteorder()
+            ret = ret.view(ret.dtype.newbyteorder()).byteswap()

```

Comment:
This appears to be the only actual code change needed to support numpy 2.

Reply:
Thanks for catching it!